### ISSUE TITLE
forking

### ISSUE THREAD

**Issue by uzleosharif** [Minres/TGC-VP#8]
Hi cloning this repo as is (https://github.com/Minres/TGC-VP.git) works fine as intended though,
however, if we try to fork this repo (e.g. like in VP-Vibes repo) and then try to clone 
```
$ git clone --recursive -b develop <fork-repo-url> 
```
then the clone process breaks at some point asking for username/password access for https://git.minres.com which fails for non Minres access. Would be nice to be able to fork this repo and able to contribute externally

i provide log for clone process in following:

```
$ git clone --recursive https://github.com/uzleosharif/TGC-VP.git
Cloning into 'TGC-VP'...
remote: Enumerating objects: 132, done.
remote: Counting objects: 100% (132/132), done.
remote: Compressing objects: 100% (113/113), done.
remote: Total 132 (delta 14), reused 129 (delta 11), pack-reused 0
Receiving objects: 100% (132/132), 74.11 KiB | 4.63 MiB/s, done.
Resolving deltas: 100% (14/14), done.
Submodule 'scc' (https://github.com/VP-Vibes/SystemC-Components.git) registered for path 'scc'
Submodule 'tgc-iss/dbt-rise-core' (https://github.com/Minres/DBT-RISE-Core.git) registered for path 'tgc-iss/dbt-rise-core'
Submodule 'tgc-iss/dbt-rise-tgc' (https://git.minres.com/DBT-RISE/DBT-RISE-TGC.git) registered for path 'tgc-iss/dbt-rise-tgc'
Submodule 'vpvper' (https://github.com/VP-Vibes/VPV-Peripherals.git) registered for path 'vpvper'
Cloning into '/tmp/TGC-VP/scc'...
remote: Enumerating objects: 9943, done.        
remote: Counting objects: 100% (553/553), done.        
remote: Compressing objects: 100% (351/351), done.        
remote: Total 9943 (delta 308), reused 376 (delta 189), pack-reused 9390        
Receiving objects: 100% (9943/9943), 10.94 MiB | 11.60 MiB/s, done.
Resolving deltas: 100% (6517/6517), done.
Cloning into '/tmp/TGC-VP/tgc-iss/dbt-rise-core'...
remote: Enumerating objects: 922, done.        
remote: Counting objects: 100% (359/359), done.        
remote: Compressing objects: 100% (159/159), done.        
remote: Total 922 (delta 202), reused 324 (delta 185), pack-reused 563        
Receiving objects: 100% (922/922), 265.16 KiB | 3.01 MiB/s, done.
Resolving deltas: 100% (594/594), done.
Cloning into '/tmp/TGC-VP/tgc-iss/dbt-rise-tgc'...
remote: Enumerating objects: 3958, done.        
remote: Counting objects: 100% (3958/3958), done.        
remote: Compressing objects: 100% (1978/1978), done.        
remote: Total 3958 (delta 2590), reused 2599 (delta 1809), pack-reused 0
Receiving objects: 100% (3958/3958), 2.87 MiB | 8.81 MiB/s, done.
Resolving deltas: 100% (2590/2590), done.
Cloning into '/tmp/TGC-VP/vpvper'...
remote: Enumerating objects: 517, done.        
remote: Counting objects: 100% (517/517), done.        
remote: Compressing objects: 100% (407/407), done.        
remote: Total 517 (delta 369), reused 246 (delta 108), pack-reused 0        
Receiving objects: 100% (517/517), 174.72 KiB | 4.99 MiB/s, done.
Resolving deltas: 100% (369/369), done.
Submodule path 'scc': checked out '1b3e1b3efd50ce7114a20c8da9245a7ee1866876'
Submodule 'axi_chi' (https://github.com/Arteris-IP/tlm2-interfaces.git) registered for path 'scc/third_party/axi_chi'
Cloning into '/tmp/TGC-VP/scc/third_party/axi_chi'...
remote: Enumerating objects: 1044, done.        
remote: Counting objects: 100% (1044/1044), done.        
remote: Compressing objects: 100% (745/745), done.        
remote: Total 1044 (delta 654), reused 669 (delta 289), pack-reused 0        
Receiving objects: 100% (1044/1044), 1.90 MiB | 11.76 MiB/s, done.
Resolving deltas: 100% (654/654), done.
Submodule path 'scc/third_party/axi_chi': checked out '50689e297706fd3ef7e0f93d7c16afe237b5a684'
Submodule path 'tgc-iss/dbt-rise-core': checked out 'fd1a504ac63b330f6214710e9259637c3e9ef524'
Submodule path 'tgc-iss/dbt-rise-tgc': checked out '09b01af3fa117e07c665f29b77b3771547771ac2'
Submodule 'gen_input/CoreDSL-Instruction-Set-Description' (https://git.minres.com/DBT-RISE/CoreDSL-Instruction-Set-Description.git) registered for path 'tgc-iss/dbt-rise-tgc/gen_input/CoreDSL-Instruction-Set-Description'
Cloning into '/tmp/TGC-VP/tgc-iss/dbt-rise-tgc/gen_input/CoreDSL-Instruction-Set-Description'...
Username for 'https://git.minres.com': uzleosharif
Password for 'https://uzleosharif@git.minres.com': warning: Clone succeeded, but checkout failed.

```

**Comment by eyck** [comment]
The problem is that you try to clone the main branch which was not up-to-date with the develop branch and pointed to an obsolete repo.
I fixed that with #9 so please update your fork.

**Comment by uzleosharif** [comment]
super thanks..now works fine
